### ISSUE TITLE
Dockerfile RUN with host local script

### ISSUE THREAD

**Issue by computerquip** [docker/buildx#918]
Something I enjoy with buildah is that when `buildah run` is called, it binds stdin in a way that I can do something like `bash -s < my_local_script`. Ultimately, this can be covered up even more so you can make a simple function that looks like `run-in-container "$ctr" my_local_script.sh` which would take a script on the host and pipe it to the container via stdin to run in a shell.

I can't do this within a Dockerfile. So when creating Dockerfiles, people end up having large chunks of commands that are concatenated together with `\` inside the Dockerfile itself. I don't like this simply because it doesn't work with linting utilities for shell scripts and because it makes the Dockerfile itself fairly large sometimes.

I'm aware that I can copy it over, execute it, and then delete it but this seems like it's not ideal. There might already be a way to do what I'm wanting that I'm not aware of as well.

TLDR; Is there a simple way to reference a script that you want to run in a container that's stored on the host inside of a Dockerfile?

**Comment by computerquip-work** [comment]
Good enough for me.
